includes/class-wc-subscriptions-core-plugin.php

```diff
@@ -46,6 +46,16 @@ class WC_Subscriptions_Core_Plugin {
 	 */
 	protected static $instance = null;
 
+	/**
+	 * An array of cart handler objects.
+	 *
+	 * The key is the cart type ('renewal', 'resubscribe', 'initial', 'switch') and the value is the cart handler instances (WCS_Cart_Renewal, WCS_Cart_Resubscribe and WCS_Cart_Initial_Payment etc).
+	 * Use WC_Subscriptions_Core_Plugin::instance()->get_cart_handler_instance() to get a cart handler instance.
```

Comment:
```suggestion
	 * The key is the class name of the cart handler and the value is the cart handler instances (WCS_Cart_Renewal, WCS_Cart_Resubscribe and WCS_Cart_Initial_Payment etc).
	 * Use WC_Subscriptions_Core_Plugin::instance()->get_cart_handler() to get a cart handler instance.
```

Reply:
Hmm I thought I had gotten rid of this section. I can't find the commit now but I'll fix this up too.

Reply:
Oh it was in https://github.com/Automattic/woocommerce-subscriptions-core/pull/587/commits/8d1f929d157822e2b645981c76f1d93132754fdd so probably staged but never committed.